administration/src/mui-modules/base/__tests__/FormAlert.test.tsx

```diff
@@ -0,0 +1,24 @@
+import { render } from '@testing-library/react'
+import React from 'react'
+
+import FormAlert from '../FormAlert'
+
+describe('FormAlert', () => {
+  it('should not render when errorMessage is null', () => {
+    const { container } = render(<FormAlert errorMessage={null} />)
+    expect(container.firstChild).toBeNull()
+  })
+
+  it('should not render when errorMessage is undefined', () => {
+    const { container } = render(<FormAlert errorMessage={undefined} />)
+    expect(container.firstChild).toBeNull()
+  })
```

Comment:
Do we need to test both (null and undefined)? both are falsy anyway. 🤔

Reply:
you are right. i will change that